src/main.go

```diff
@@ -3,12 +3,12 @@ package main
 import (
 	"log"
 	"net/http"
+	"src/services/palindrome"
 
-	"github.com/oreuta/elementary-service/src/services/squarert"
 )
 
 func main() {
 	log.Println("Elementary-service started...")
-	http.HandleFunc("/squarert", squarert.Handler)
```

Comment:
i suppose we dont need to delete this handler, because its still at the project:)

Reply:
do u think we will be presenting SquareRoot function on Wednesday?